keyboards/skyloong/gk61/pro/info.json

```diff
@@ -0,0 +1,352 @@
+            {"matrix": [1, 9], "x": 153, "y": 13, "flags": 4},
+            {"matrix": [1, 10], "x": 170, "y": 13, "flags": 4},
+            {"matrix": [1, 11], "x": 186, "y": 13, "flags": 4},
+            {"matrix": [1, 12], "x": 202, "y": 13, "flags": 4},
+            {"matrix": [1, 13], "x": 222, "y": 13, "flags": 4},
+            {"matrix": [2, 0], "x": 6, "y": 26, "flags": 4},
+            {"matrix": [2, 1], "x": 28, "y": 26, "flags": 4},
+            {"matrix": [2, 2], "x": 44, "y": 26, "flags": 4},
+            {"matrix": [2, 3], "x": 61, "y": 26, "flags": 4},
+            {"matrix": [2, 4], "x": 77, "y": 26, "flags": 4},
+            {"matrix": [2, 5], "x": 93, "y": 26, "flags": 4},
+            {"matrix": [2, 6], "x": 109, "y": 26, "flags": 4},
+            {"matrix": [2, 7], "x": 125, "y": 26, "flags": 4},
+            {"matrix": [2, 8], "x": 141, "y": 26, "flags": 4},
+            {"matrix": [2, 9], "x": 157, "y": 26, "flags": 4},
+            {"matrix": [2, 10], "x": 174, "y": 26, "flags": 4},
+            {"matrix": [2, 11], "x": 190, "y": 26, "flags": 4},
+            {"matrix": [2, 13], "x": 216, "y": 26, "flags": 4},
+            {"matrix": [3, 0], "x": 10, "y": 51, "flags": 4},
+            {"matrix": [3, 1], "x": 36, "y": 51, "flags": 4},
+            {"matrix": [3, 2], "x": 53, "y": 51, "flags": 4},
+            {"matrix": [3, 3], "x": 69, "y": 51, "flags": 4},
+            {"matrix": [3, 4], "x": 85, "y": 51, "flags": 4},
+            {"matrix": [3, 5], "x": 101, "y": 51, "flags": 4},
+            {"matrix": [3, 6], "x": 117, "y": 51, "flags": 4},
+            {"matrix": [3, 7], "x": 133, "y": 51, "flags": 4},
+            {"matrix": [3, 8], "x": 149, "y": 51, "flags": 4},
+            {"matrix": [3, 9], "x": 166, "y": 51, "flags": 4},
+            {"matrix": [3, 10], "x": 182, "y": 51, "flags": 4},
+            {"matrix": [3, 12], "x": 212, "y": 51, "flags": 4},
+            {"matrix": [4, 0], "x": 2, "y": 64, "flags": 4},
+            {"matrix": [4, 1], "x": 22, "y": 64, "flags": 4},
+            {"matrix": [4, 2], "x": 42, "y": 64, "flags": 4},
+            {"matrix": [4, 4], "x": 75, "y": 64, "flags": 4},
+            {"matrix": [4, 5], "x": 103, "y": 64, "flags": 4},
+            {"matrix": [4, 6], "x": 103, "y": 64, "flags": 4},
+            {"matrix": [4, 8], "x": 131, "y": 64, "flags": 4},
+            {"matrix": [4, 9], "x": 163, "y": 64, "flags": 4},
+            {"matrix": [4, 10], "x": 184, "y": 64, "flags": 4},
+            {"matrix": [4, 11], "x": 204, "y": 64, "flags": 4},
+            {"matrix": [4, 13], "x": 224, "y": 64, "flags": 4}
+        ],
+        "max_brightness": 200,
+        "hue_steps": 20,
+        "sat_steps": 20,
+        "speed_steps": 20,
+        "val_steps": 20
+    },
+    "url": "https://github.com/JZ-Skyloong",
+    "usb": {
+        "device_version": "1.0.0",
+        "force_nkro": true,
+        "pid": "0x6A61",
+        "vid": "0x1EA7",
```

Comment:
This vendor ID looks to belong to Sharkoon
https://www.the-sz.com/products/usbid/index.php?v=0x1EA7&p=&n=

Please change the Vendor ID.  Anything past 0x4000 is generally unassigned, but the further past, the better, as it will be loger to be officially assigned.

Reply:
This VID is registered by SEMITEK INTERNATIONAL (HK) HOLDING LTD. on USB org(https://www.usb.org/sites/default/files/vendor_ids042523.pdf). Skyloong is a subsidiary of semitek. Sharkoon has used the VID without our authorization and we have sent a letter to them asking them to stop using it.